.goreleaser.yml

```diff
@@ -4,16 +4,19 @@ env:
   # TODO (https://github.com/infrahq/infra/issues/65)
   - CGO_ENABLED=0
   - GO111MODULE=on
+  - BUILDVERSION_PRERELEASE={{ if index .Env "BUILDVERSION_PRERELEASE" }}{{ .Env.BUILDVERSION_PRERELEASE }}{{ end }}
+  - BUILDVERSION_METADATA={{ if index .Env "BUILDVERSION_METADATA" }}{{ .Env.BUILDVERSION_METADATA }}{{ end }}
```

Comment:
For my own understanding, why is this necessary? It's not possible to reference the env directly below unless we set it here?

Reply:
This is setting a default. goreleaser will error if the var isn't set. The unspoken part of this if is the else which is an empty string